phoenix-core/src/main/java/org/apache/phoenix/compile/ExpressionCompiler.java

```diff
@@ -1339,4 +1362,193 @@ public Expression visit(SubqueryParseNode node) throws SQLException {
     public int getTotalNodeCount() {
         return totalNodeCount;
     }
+  //JSON node 
+    @Override
+    public boolean visitEnter(JsonSingleKeySearchParseNode node) {
+        return true;
+    }
+    
+    @Override
+    public boolean visitEnter(JsonContainWithinRightParseNode node) {
+        return true;
+    }
+    
+    @Override
+    public boolean visitEnter(JsonContainWithinLeftParseNode node) {
+        return true;
+    }
+    
+    @Override
+    public boolean visitEnter(JsonMultiKeySearchOrParseNode node) {
+        return true;
+    }
+    
+    @Override
+    public boolean visitEnter(JsonMultiKeySearchAndParseNode node) {
+        return true;
+    }
+    
+    @Override
+    public boolean visitEnter(JsonPathAsTextParseNode node) {
+        return true;
+    }
+    
+    @Override
+    public boolean visitEnter(JsonPathAsElementParseNode node) {
+        return true;
+    }
+    
+    @Override
+    public boolean visitEnter(JsonPointAsElementParseNode node) throws SQLException {
+        return true;
+    }
+    
+    @Override
+    public boolean visitEnter(JsonPointAsTextParseNode node){
+    	return true;
+    }
+    
+    
+    @Override
+    public Expression visitLeave(JsonSingleKeySearchParseNode node, List<Expression> children) throws SQLException {
+        ParseNode rhsNode = node.getChildren().get(1);
```

Comment:
Sorry,it is our mistake.
We will remove those checking on next update.
Thank @twdsilva